src/main/java/by/itacademy/javaenterprise/knyazev/entities/Teacher.java

```diff
@@ -0,0 +1,30 @@
+package by.itacademy.javaenterprise.knyazev.entities;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+
+import lombok.Data;
+
+@Data
+@Entity
+public class Teacher {
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+	@Column(nullable = false)
+	private String name;
+	@Column(nullable = false)
+	private int age;
+	
+	@OneToOne(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
```

Comment:
```suggestion
	@OneToOne(fetch = FetchType.EAGER, cascade = CascadeType.ALL)
```
`LAZY` для 1к1 нужен в очень редких случаях